src/Utils/RMC/RMST.as

```diff
@@ -448,11 +439,18 @@ class RMTS : RMS {
 
     void RenderBrokenButton() {
         if (UI::OrangeButton(Icons::PlayCircleO + "Skip broken Map")) {
-            RMC::PauseRun();
-              RMC::TimePaused += RMC::TimeSpentMap;
-            Log::Trace(GetModeNameShort() + ": Skipping broken map");
+            if (!UI::IsOverlayShown()) UI::ShowOverlay();
+            RMC::IsPaused = true;
+            RMC::ClickedOnSkip = true;
+            // XXX implement an RMST version of this modal? It currently triggers SwitchMap which cause a local map switch (see BrokenMapSkipWarnModalDialog.as)
+            //Renderables::Add(BrokenMapSkipWarnModalDialog());
+            // Below logic is copypasted from the above commented-out function.
```

Comment:
Is/Would the function be called anywhere else in this class? If not, I'd say this is perfectly fine, assuming it was copy/pasted to replace text/icons. The comment might be fine to remove. It can be called out in PR review when we seek to merge this.